method-proposals/PROPOSAL-did-cheqd.md

```diff
@@ -0,0 +1,142 @@
+* Deactivation of DIDs supported
+* Full support for verification method relationships, whereby different key types can be specified for defined purposes within the DID Document, including `assertionMethod`, `keyAgreement` `capabilityInvocation` and `capabilityDelegation`, including both **referenced** and **embedded** keys as defined in [DID Core](https://www.w3.org/TR/did-1.0/)
+* `alsoKnownAs` allows synonymous DIDs to be published in alternative locations, such as on a web server using `did:web` or `did:webvh`
+* [DID-Linked Resources](https://w3c-ccg.github.io/DID-Linked-Resources/) supported, with optional inclusion in the DID Document Metadata, to create searchable directories of DIDs and associated collections of resources
+* Support for multiple namespaces, including `did:cheqd:testnet` for testing, as well as for `did:cheqd:mainnet` mainnet for full, consensus-backed security
+* Transactions for writing `did:cheqd` DIDs in MiCA Regulated Electronic Money Tokens (EMTs), including USDC and EURe
+* Clear and consistent pricing for DIDs and DID-Linked Resources
+* Decentralized governance framework for updates to software updates or pricing changes for DID Method on cheqd Network, voted through by the validator nodes and delegators who stake the $CHEQ token
+* Innovative trust registry / trust infrastructure model natively supported, with hierarchical DID and VC relationships able to be expressed to mirror traditional PKI models that would usually adopt X.509 certificates
+* Support in 5+ Open Source SDKs and libraries
+* Support in both Universal Resolver and Universal Registrar
+
+### Drawbacks
+
+* Token used for underlying transactions ($CHEQ), which may limit adoption owing to lack of regulatory clarity in certain jurisdictions. Although this drawback is mitigated through the support for MiCA regulated stablecoins natively on the network.
+* Volatile transaction pricing for DIDs, based on fluctuations in value of $CHEQ.
+* Immutability of DIDs, which may be problematic if sensistive data such as Personally Identifiable Information (PII) is pointed to within the `serviceEndpoint` or as a DID-Linked Resource.
+
+## Existing materials
+
+- [DID Method Specification](https://docs.cheqd.io/product/architecture/adr-list/adr-001-cheqd-did-method)
+- Implementations
+  - [Veramo SDK plugin](https://docs.cheqd.io/product/sdk/veramo-plugin)
+  - [Credo](https://credo.js.org/guides/getting-started/set-up/cheqd)
+  - [Aries Cloud Agent Python (ACA-Py)](https://docs.cheqd.io/product/sdk/aca-py)
+  - [Dock SDK](https://github.com/docknetwork/sdk)
+  - [Walt.id Community Stack](https://walt.id/blog/p/community-stack)
+  - [cheqd Studio](https://cheqd.io/solutions/cheqd-studio/)
+  - [Universal Registrar Driver](https://github.com/decentralized-identity/universal-registrar)
+  - [Universal Resolver Driver](https://github.com/decentralized-identity/universal-resolver)
+  - [Vidos Universal Resolver](https://vidos.id/docs/services/resolver)
```

Comment:
Don't this is independent, just an instance of the one created by cheqd?

Reply:
Not sure what you mean here

Reply:
I mean that Vidos is just running an instance cheqd's resolver driver / Universal Resolver. It's not a separate implementation.

Reply:
Changed